com.vrcfury.vrcfury/Editor/VF/Inspector/VRCFuryActionEditor.cs

```diff
@@ -108,6 +108,126 @@ public class VRCFuryActionDrawer : PropertyDrawer {
+                var propField = VRCFuryEditorUtils.Prop(rendererProp);
+                propField.style.flexGrow = 1;
+                propField.style.flexShrink = 1;
+                propField.SetEnabled(!affectAllMeshesProp.boolValue);
+                rendererRow.Add(propField);
+                
+                rendererRow.Add(new Label("All Meshes") {
+                    style = {
+                        marginLeft = 2,
+                        marginRight = 2,
+                        flexGrow = 1,
+                        flexBasis = 100,
+                        unityTextAlign = TextAnchor.MiddleRight
+                    }
+                });
+                
+                var propField4 = VRCFuryEditorUtils.Prop(affectAllMeshesProp);
+                propField4.style.flexGrow = 0;
+                propField4.style.flexShrink = 0;
+                propField4.style.flexBasis = 16;
+                propField4.schedule.Execute(() => {
+                    propField.SetEnabled(!affectAllMeshesProp.boolValue);
+                }).Every(1000);
+                rendererRow.Add(propField4);
+                
+                col.Add(rendererRow);
+                
+                var materialRow = new VisualElement {
+                    style = {
+                        flexDirection = FlexDirection.Row,
+                        alignItems = Align.FlexStart
+                    }
+                };
+            
+                var propertyNameProp = prop.FindPropertyRelative("propertyName");
+                var propField2 = VRCFuryEditorUtils.Prop(propertyNameProp);
+                propField2.style.flexGrow = 1;
+                propField2.tooltip = "Property Name";
+                materialRow.Add(propField2);
+                
+                var propField3 = VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("value"));
+                propField3.style.flexGrow = 0;
+                propField3.style.flexBasis = 60;
+                propField3.tooltip = "Property Value";
+                materialRow.Add(propField3);
+
+                var searchButton = new Button(SearchClick)
+                {
+                    text = "Search",
+                    style =
+                    {
+                        marginTop = 0,
+                        marginLeft = 0,
+                        marginRight = 0,
+                        marginBottom = 0
+                    }
+                };
+                materialRow.Add(searchButton);
+                col.Add(materialRow);
+                
+                row.Add(col);
+
+                return row;
+
+                void SearchClick() {
+                    var targetWidth = row.GetFirstAncestorOfType<UnityEditor.UIElements.InspectorElement>().worldBound
+                        .width;
+                    var searchContext = new UnityEditor.Experimental.GraphView.SearchWindowContext(GUIUtility.GUIToScreenPoint(Event.current.mousePosition), targetWidth, 300);

```

Comment:
Would not be surprised if this breaks during the 2022 editor upgrade. They moved a lot of stuff out of Experimental.

Reply:
Apparently it didn't move. Nice.